src/components/Dashboard.tsx

```diff
@@ -66,9 +69,15 @@ const TemplateComponent: React.FC<{ template: TemplateInList }> = ({ template })
               {template?.namePlural || `${name} ${strings.LABEL_APPLICATIONS}`}
             </a>
             <Icon name="chevron right" />
+            {loadedFiltersCount !== filters.length && <LoadingSmall />}
           </Label>
-          {filters.map((filter) => (
-            <FilterComponent key={filter.id} template={template} filter={filter} />
+          {filters.map((filter: { id: any }) => (
```

Comment:
```suggestion
          {filters.map((filter) => (
```
This doesn't actually compile like this, as the destructuring makes the parent "filter" not available, but "filter" is required for the filter prop. (And you're not actually using this id anywhere)